crates/forge_main/src/ui.rs

```diff
@@ -129,6 +129,38 @@ impl<F: API> UI<F> {
 
                     input = self.console.prompt(None).await?;
                 }
+                Command::Retry => {
+                    if let Some(conversation_id) = self.state.conversation_id.clone() {
+                        CONSOLE.writeln(
+                            TitleFormat::execute("retry")
+                                .sub_title(self.state.usage.to_string())
+                                .format(),
+                        )?;
```

Comment:
I think we can drop this message. It's a bit noisy.

Reply:
Understood.